src/components/layout/footer.tsx

```
@@ -0,0 +1,74 @@
+
+        return null
+    } catch (e) {
+        console.error('Failed to fetch latest commit', e)
+        return null
+    }
+}
+
+export function Footer() {
+    const [commit] = createResource(fetchLatestCommit)
+
+    const version = () => {
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(indent)